Vulkan-Pipeline-Assistant/Vulkan/vulkanmain.cpp

```diff
@@ -21,6 +24,26 @@ namespace vpa {
         if(m_main->m_currentState == VulkanState::Ok && !m_handlingResize) m_main->RecreateSwapchain();
     }
 
+    bool VulkanWindow::nativeEvent(const QByteArray &eventType, void* message, long* result) {
+        Q_UNUSED(result)
+        Q_UNUSED(eventType)
+        if (QGuiApplication::platformName() == "windows") return WindowsNativeEvent(static_cast<MSG*>(message));
```

Comment:
`if (QGuiApplication::platformName() == "windows") return WindowsNativeEvent(static_cast<MSG*>(message));`

can be 

`return (QGuiApplication::platformName() == "windows") && WindowsNativeEvent(static_cast<MSG*>(message));`